tokens/tokens-eth.json

```diff
@@ -7135,6 +7135,38 @@
   "youtube"   : ""
 }
 },{
+"symbol"      : "RIPT",
+"address"     : "0xdd007278b667f6bef52fd0a4c23604aa1f96039a",
+"decimals"    : "8",
+"name"        : "RIPTIDECOIN",
+"ens_address" : "",
+"website"     : "https://riptidecoin.com/",
```

Comment:
would remove the last slash

Reply:
do we need to make the changes and re pull?

Reply:
done thanks.